docs/tutorial/GetStarted.md

```diff
@@ -0,0 +1,386 @@
+		- [Test your container](#test-your-container)
+		- [Tag and publish your container](#tag-and-publish-your-container)
+	- [Step 3 : Creating a DC/OS Package](#step-3-creating-a-dcos-package)
+		- [config.json](#configjson)
+		- [resource.json](#resourcejson)
+		- [package.json](#packagejson)
+		- [marathon.json.mustache](#marathonjsonmustache)
+	- [Step 4 : Testing the package](#step-4-testing-the-package)
+		- [Validation using build script.](#validation-using-build-script)
+		- [Build the local Universe server](#build-the-local-universe-server)
+		- [Run the local Universe server](#run-the-local-universe-server)
+		- [Add the Universe repo to DC/OS cluster:](#add-the-universe-repo-to-dcos-cluster)
+		- [Install the package](#install-the-package)
+	- [Step 5 : Publish the package](#step-5-publish-the-package)
+
+## Required nomenclature
+This guide assumes you are familiar with the basics of the concepts mentioned below. Advanced users can skip this section and jump to [Create a package](#create-a-package). If you are new to DC/OS, we recommend reading this section along with the external links provided.
+
+
+### What is Universe?
+The Universe is a DC/OS package repository that contains services like Spark, Cassandra, Jenkins, and many others. It allows users to install these services with a single click from the DC/OS UI or by a simple `dcos package install <package_name>` command from the DC/OS CLI. Many community members have already submitted their own packages to the Universe, and we encourage anyone interested to get involved with package development! It is a great way of contributing to the DC/OS ecosystem and allows users to easily get started with your favorite package.
+
+
+### What is Marathon?
+[Marathon](https://mesosphere.github.io/marathon/) is a production-grade container orchestration platform for Mesosphere’s Datacenter Operating System (DC/OS) and [Apache Mesos](https://mesos.apache.org/). In order to deploy applications on top of Mesos, one can use Marathon for Mesos. Marathon is a cluster-wide init and control system for running Linux services in cgroups and Docker containers. Marathon has a number of different deploy [features](https://mesosphere.github.io/marathon/#features) and is a very mature project. Marathon runs on top of Mesos, which is a highly scalable, battle tested and flexible resource manager. Marathon is proven to scale and runs in many production environments.
+
+
+### What is a package?
+There are several ways to deploy your service onto a running DC/OS cluster.
+  * Use the DC/OS Marathon command in the CLI,
+  * Use the Marathon REST API directly, or
+  * Deploy your service as a package.
+
+Deploying your service using the package approach makes your life easier and service management efficient. Once you have a running DC/OS cluster, you will be able to browse packages in the dashboard. A package consists of the four required configuration files and all of the external files linked from them.
```

Comment:
> A package consists of the four required configuration files ...

Since you mention a few of these files by name in the "Prerequisites" section below, it might be a good idea to name them here so that readers know that they are part of a package.